test/options/lines-between-rulesets-css/test.js

```diff
@@ -0,0 +1,24 @@
+/* global describe, it */
+'use strict';
+
+describe('options/lines-between-rulesets:css', function() {
```

Comment:
Fair point. Not sure if there's an easy way for me to jump in on this PR to help—I guess I'll wait a bit to see if @AoDev has time to address this before creating my own fork…